db/migrate/20211013035935_add_updated_at_to_stage_and_tournament.rb

```diff
@@ -0,0 +1,6 @@
+class AddUpdatedAtToStageAndTournament < ActiveRecord::Migration[5.2]
+  def change
+    add_timestamps :stages, null: false, default: -> { 'NOW()' }
+    add_column :tournaments, :updated_at, :datetime
```

Comment:
This should probably also be `null: false, default: -> { 'NOW()' }

Reply:
FWIW no tournaments on the server have non-null created_at timestamps so i am gonna leave that one as is for now.

Reply:
Added the defaults for the new column.